pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -203,6 +203,52 @@ func NewPolicyApplier(rootNamespace string,
 	}
 }
 
+type RemoteJwksMode int
+
+const (
+	// Istiod is used to indicate Istiod ALWAYS fetches the JWKs server
+	Istiod RemoteJwksMode = iota
+
+	// Hybrid is used to indicate Envoy fetches the JWKs server when there is a cluster entry,
+	// otherwise fallback to Istiod
+	Hybrid
+
+	// Envoy is used to indicate Envoy ALWAYS fetches the JWKs server
+	Envoy
+)
+
+// String converts RemoteJwksMode to readable string.
+func (mode RemoteJwksMode) String() string {
```

Comment:
Do we want to continue to call it RemoteJwks? We are basically specifying which entity is fetching JWKs. Maybe "JwksFetcher" (any better name suggestion?) is more intuitive?

Reply:
how about FetchJwksMode?